File: Hospital-MS/Hospital-MS.Core/Common/PagingFilterModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Hospital_MS.Core.Common
{
    public class PagingFilterModel
    {
        public string SearchText { get; set; }
        public int CurrentPage { get; set; }
        public int PageSize { get; set; }
        public List<FilterModel> FilterList { get; set; }
    }
}

File: Hospital-MS/HMS.Client/src/app/Models/Generics/PagingFilterModel.ts

export interface PagingFilterModel {
    searchText?: string;
    currentPage: number;
    pageSize: number;
    filterList: FilterModel[];
    filterType?: string;
    filterItems?: string[]
}


export interface FilterModel {
    categoryName: string;
    itemId?: string;
    itemKey?: string;
    itemValue?: string;
    isChecked?: boolean;
    fromDate?: string | null;
    toDate?: string | null;
    filterType?: string;
    isVisible?: boolean;
    filterItems?: FilterModel[];
    displayOrder?: number;
}